<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EC2AMAZ-AD3GHSU" ignored="0" name="FirefoxTest" tests="4" failures="2" timestamp="03 Sep 2018 11:48:17 GMT" time="27.173" errors="0">
  <testcase name="@BeforeClass launchBrowser" time="24.672" classname="com.test.FirstTest">
    <failure type="org.openqa.selenium.WebDriverException" message="connection reset
Build info: version: &amp;apos;3.14.0&amp;apos;, revision: &amp;apos;aacccce0&amp;apos;, time: &amp;apos;2018-08-02T20:19:58.91Z&amp;apos;
System info: host: &amp;apos;EC2AMAZ-AD3GHSU&amp;apos;, ip: &amp;apos;172.31.40.111&amp;apos;, os.name: &amp;apos;Windows Server 2016&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_181&amp;apos;
Driver info: driver.version: FirefoxDriver
remote stacktrace: ">
      <![CDATA[org.openqa.selenium.WebDriverException: connection reset
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'EC2AMAZ-AD3GHSU', ip: '172.31.40.111', os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: driver.version: FirefoxDriver
remote stacktrace: 
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$new$0(W3CHandshakeResponse.java:57)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$getResponseFunction$2(W3CHandshakeResponse.java:104)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:122)
at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:125)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:212)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:125)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:103)
at com.test.FirstTest.launchBrowser(FirstTest.java:23)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass launchBrowser -->
  <testcase name="tc_1" time="0.0" classname="com.test.FirstTest">
    <skipped/>
  </testcase> <!-- tc_1 -->
  <testcase name="tc_2" time="0.0" classname="com.test.FirstTest">
    <skipped/>
  </testcase> <!-- tc_2 -->
  <testcase name="tc_3" time="2.017" classname="com.test.SecondTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.test.SecondTest.tc_3(SecondTest.java:14)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- tc_3 -->
  <testcase name="tc_3" time="2.039" classname="com.test.SecondTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.test.CustomListener.onTestFailure(CustomListener.java:20)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- tc_3 -->
</testsuite> <!-- FirefoxTest -->
